src/Icon/README.md

```diff
@@ -1 +1,14 @@
 # Icon
+
+The Icon component is actually an object which holds all Icon components. 
+Trying to render `<Icon />` will result in an error, but for example, `<Icon.ArrowBack />` would not.
+
+## Usage
```

Comment:
As they're all the same, I'm kosher with this.

Reply:
I was considering providing a second example, but for right now I'm only exporting `Icon` to be used. Since there is some uncertainty as to whether using `<Icon.Whatever /> ` will tree shake, I will go back and update it so that all Icons are exported individually. It will make our main index.ts file gross, but best to hedge our bets.